### ISSUE TITLE
keep_single_parameter_on_one_line, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#973]
it should be okay to split if its a VALUE

code

```abap
lo_bar->call( VALUE #(
  foo = lv_foo
  bar = lv_bar ) ).
```

example

![image](https://user-images.githubusercontent.com/5888506/84862724-f8caf680-b073-11ea-8e59-40a945936dc4.png)
